### PR TITLE
Constants fix

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
Ready

## Description
old format (unified yml) integrations and scripts were not marked as changed.

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

File: Tests/scripts/validate_files.py [hunk 1]
```diff
@@ -118,14 +118,17 @@ def get_modified_files(files_string, tag='master', print_ignored_files=False):
             elif file_status.lower() == 'd' and checked_type(file_path) and not file_path.startswith('.'):
                 deleted_files.add(file_path)
             elif file_status.lower().startswith('r') and checked_type(file_path):
-                modified_files_list.add((file_data[1], file_data[2]))
+                if checked_type(file_data[2], CODE_FILES_REGEX):
```
Comment (Itay4) [inline Tests/scripts/validate_files.py]: add comment

File: Tests/scripts/infrastructure_tests/validate_files_test.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+def packagify_mock_no_change(modified, added, removed, tag):
+    return modified, added, removed
+
+
+def test_get_modified_files(mocker):
+    mocker.patch('Tests.scripts.hook_validations.conf_json.ConfJsonValidator.load_conf_file', return_value={})
+    file_validator = FilesValidator()
+
+    changed_files = '''A       Integrations/Recorded_Future/CHANGELOG.md
```
Comment (Itay4) [inline Tests/scripts/infrastructure_tests/validate_files_test.py]: add test to check changed yml
add test to packagify

Comment (yaakovi) [reply Tests/scripts/infrastructure_tests/validate_files_test.py]: done

File: Tests/scripts/validate_files.py (hunk 1 above)
Comment (yaakovi) [reply Tests/scripts/validate_files.py]: done

**Pull request merged**